src/torcontrol.cpp

```diff
@@ -409,7 +409,7 @@ static bool WriteBinaryFile(const fs::path &filename, const std::string &data)
 class TorController
 {
 public:
-    TorController(struct event_base* base, const std::string& target);
+    TorController(struct event_base* base, const std::string& target, uint16_t default_port);
```

Comment:
In commit "tor: pass in Tor control port rather than using globals" (8f9062a84d01a709d9303d21f25e244a1dbd0044)

It seems like you should `s/default_port/listen_port/` everywhere in this commit. If I understand correctly, the value starts off being called `listen_port` in `AppInitMain`, then is called `default_port` in `StartTorControl`, then is called `m_listen_port` again in `TorController`. `default_port` just seems like a seems like confusing and undescriptive name here.